darshan-util/pydarshan/darshan/experimental/plots/plot_access_histogram.py

```diff
@@ -26,41 +23,20 @@ def plot_access_histogram(report, mod, ax=None):
     if 'mod_agg_iohist' in dir(report):
         report.mod_agg_iohist(mod)
     else:
-        print("Can not create summary, mod_agg_iohist aggregator is not registered with the report class.")
+        print(
+            "Cannot create summary, mod_agg_iohist aggregator"
+            "is not registered with the report class."
+        )
 
     # defaults
-    labels = ['0-100', '101-1K', '1K-10K', '10K-100K', '100K-1M', '1M-4M', '4M-10M', '10M-100M', '100M-1G', '1G+']
-    read_vals = [0, 0, 0, 0, 0,  0, 0, 0, 0, 0]
-    write_vals = [0, 0, 0, 0, 0,  0, 0, 0, 0, 0]
-
-    agg = report.summary['agg_iohist'][mod]
-
-    read_vals = [
-        agg['READ_0_100'],
-        agg['READ_100_1K'],
-        agg['READ_1K_10K'],
-        agg['READ_10K_100K'],
-        agg['READ_100K_1M'],
-        agg['READ_1M_4M'],
-        agg['READ_4M_10M'],
-        agg['READ_10M_100M'],
-        agg['READ_100M_1G'],
-        agg['READ_1G_PLUS']
-    ]
-
-    write_vals = [
-        agg['WRITE_0_100'],
-        agg['WRITE_100_1K'],
-        agg['WRITE_1K_10K'],
-        agg['WRITE_10K_100K'],
-        agg['WRITE_100K_1M'],
-        agg['WRITE_1M_4M'],
-        agg['WRITE_4M_10M'],
-        agg['WRITE_10M_100M'],
-        agg['WRITE_100M_1G'],
-        agg['WRITE_1G_PLUS']
+    labels = [
+        '0-100', '101-1K', '1K-10K', '10K-100K', '100K-1M',
+        '1M-4M', '4M-10M', '10M-100M', '100M-1G', '1G+',
     ]
 
+    agg_vals = list(report.summary['agg_iohist'][mod].values())
+    read_vals = agg_vals[:10]
+    write_vals = agg_vals[10:]
```

Comment:
It looks like you're depending on the insertion order of the objects in the dictionary being preserved, but we need to support Python 3.6 for the moment, for which this is not considered good practice because it is an implementation detail not guaranteed until Python 3.7.

So, I would just add a small comment or TODO to the code and leave this alone for now until we're at higher minimum Python version. You could still delete the pointless double assignment to `read_vals` and `write_vals`.

Reply:
Okay, I wasn't aware of that difference so I've made the recommended changes here.